### PR TITLE
Disable L2ListenTcpdump test

### PR DESCRIPTION
**Pull request by gpotter2**
We will certainly lose a little bit of coverage, but I just got merged PRs raising it.
This test is annoying

### CODE REVIEW

File: .travis/test.sh [hunk 1]
```diff
@@ -21,13 +21,6 @@ then
     UT_FLAGS=" -K tcpdump"
   fi
 
-  if python --version 2>&1 | grep -q '^Python 3\.'
```
Comment (guedou) [inline .travis/test.sh]: Why did you remove these lines?

File: .travis/test.sh [hunk 2]
```diff
@@ -78,13 +71,6 @@ then
   UT_FLAGS+=" -K crypto -K not_pypy"
 fi
 
-if python --version 2>&1 | grep -q '^Python 3\.'
```
Comment (guedou) [inline .travis/test.sh]: Why did you remove these lines?

Comment (gpotter2) [reply .travis/test.sh]: They are only used with this test.

Comment (guedou) [reply .travis/test.sh]: I think that we can only comment the failing test and keep the tags are they are today. It will help us track the fact that we need to fix the test =)

Comment (guedou) [reply .travis/test.sh]: Could you keep the changes to the regressions.uts ?

Comment (gpotter2) [reply .travis/test.sh]: Done

File: test/regression.uts [hunk 3]
```diff
@@ -1205,24 +1205,26 @@ send_and_sniff(IP(dst="secdev.org")/ICMP(), flt="icmp")
 send_and_sniff(Ether()/IP(dst="secdev.org")/ICMP())
 
 = Test L2ListenTcpdump socket
-~ netaccess FIXME_py3
+~ netaccess
```
Comment (guedou) [inline test/regression.uts]: Keep the FIXME_py3 tag that will help us  track this test.

**Comment by guedou** [comment]
I might be responsible for these errors. The flag might not be activated for all tox builds,
for example when tox is launched using Python2 but tests Python3.
